11-React/.docker/docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
+  frontend:
+    build:
+      context: .
+      target: dev-envs
```

Comment:
why `dev-envs`?

Reply:
By specifying target: dev-envs in the build section, docker-compose will only build the frontend service's Dockerfile up to the dev-envs stage. This can save time when building the service and reduce overall build times.
also it can be remove.